src/modules/Host.rb

```diff
@@ -358,12 +358,23 @@ def set_names(address, names)
   def load_hosts
     return false if SCR.Read(path(".target.size"), CFA::Hosts::PATH) <= 0
 
-    @hosts = CFA::Hosts.new
-    @hosts.load
+    begin
```

Comment:
Did you know you can `rescue` the whole method body?
```rb
def foo
  bar
  baz
rescue E
  qux
end
```

It works just as well here, and the diff+history will be clearer.

Reply:
ok